deploy/helm/sumologic/conf/metrics/otelcol/exporters.yaml

```diff
@@ -1,3 +1,46 @@
+{{- if eq .Values.debug.metrics.metadata.print true }}
+debug:
+  verbosity: detailed
+{{- end }}
+
+{{- if eq (include "sumologic-mock.forward-metrics-metadata" .) "true" }}
+sumologic/sumologic-mock-default:
+  client: {{ include "sumologic.sumo_client" . }}
+  metric_format: {{ include "metrics.otelcol.exporter.format" . }}
```

Comment:
Is it used anywhere? In the pipeline you use only `sumologic/sumologic-mock-http`

Reply:
fixed